remotespark/livyclientlib/livysession.py

```diff
@@ -90,11 +89,14 @@ def create_sql_context(self):
         self.ipython_display.writeln("Creating HiveContext as 'sqlContext'")
         command = self._get_sql_context_creation_command()
         try:
-            command.execute(self)
+            (success, out) = command.execute(self)
         except LivyClientTimeoutError:
             raise LivyClientTimeoutError("Failed to create the SqlContext in time. Timed out after {} seconds."
                                          .format(self._wait_for_idle_timeout_seconds))
-        self.created_sql_context = True
+        if success:
```

Comment:
It is true now, but without the unit tests to all those intermediate classes, you won't know that it will never change. Once you write those tests, it would be good to say why those classes should not catch ValueErrors in the unit tests.

Please add them to prevent regressions.